internal/github/org.go

```diff
@@ -0,0 +1,18 @@
+package github
+
+import (
+	"context"
+
+	gh "github.com/google/go-github/v33/github"
+)
+
+func OrgExist(cli *gh.Client, ctx context.Context, name string) (bool, error) {
+	_, rsp, err := cli.Organizations.Get(ctx, name)
+	if rsp.StatusCode == 404 {
+		return false, nil
+	}
+	if err != nil {
+		return false, err
+	}
```

Comment:
@OrlovM in case of error, `github.OrganizationService.Get` can return `nil` response, it means that `err` should be checked for `err != nil` before using `StatusCode` of `rsp`.